src/views/Lottery/components/YourHistoryCard/FinishedRoundRow.tsx

```diff
@@ -18,6 +18,12 @@ const Grid = styled(Box)`
   cursor: pointer;
 `
 
+const StyledSmallDotIcon = styled(SmallDotIcon)`
```

Comment:
Tiny change to the icon for when a user has no winnings, or has not claimed winnings for a round